### ISSUE TITLE
One GUI using technology for web applications

### ISSUE THREAD

**Issue by hjoliver** [cylc/cylc-ui#81]
~Depends on #1872~ DONE.

**Comment by kinow** [comment]
This issue has discussions around Graph, Design, and several other topics - some covered by other issues.

I wonder if we would be able to close this one and move whatever is not documented in other issues to new, and more specific issues.

**Comment by matthewrmshin** [comment]
> I wonder if we would be able to close this one and move whatever is not documented in other issues to new, and more specific issues.

Makes sense.

**Comment by kinow** [comment]
I finished re-reading this ticket and the comments. And looks to me like we
are still in the right direction.

Most comment had features like Warning icon (already implemented in a componet
ny Oliver), GraphQL (done), sets of icons for different statuses (done but
still being discussed in other items regarding design and separation
of Job/Task), performance displaying graphs (being discussed in a separate issue
though with lower priority than the Tree view), theming (discussed in a separate issue),
accessibility (covered in design and other issues like the one for font size),


I think the only things that could be copied to other issues (or the other
issue could simply reference this one to add a link) are:

- Gantt chart sketches (https://github.com/cylc/cylc-ui/issues/81#issuecomment-420084752)
- Some comments from Hilary and Oliver about graphs and JS (https://github.com/cylc/cylc-ui/issues/81#issuecomment-423513023, https://github.com/cylc/cylc-ui/issues/81#issuecomment-423555699

Looks like most of the topics here are already being covered in separated issues :tada:  Any objection to just adding the two links above, and then closing this issue? 

And great to see that from 2017 to now, we are making good progress and in a steady trajectory :ship:

**Comment by hjoliver** [comment]
> Looks like most of the topics here are already being covered in separated issues tada Any objection to just adding the two links above, and then closing this issue?

Seems fine to me - I just quickly re-read all the comments above.

**Comment by kinow** [comment]
Comments copied to new issues. Closing this one now :+1:

**Comment by hjoliver** [comment]
Thanks @kinow :+1:
